slack-base/src/main/java/com/hubspot/slack/client/models/blocks/ActionsIF.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.List;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.elements.BlockElement;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ActionsIF extends Block {
+  String TYPE = "actions";
+
+  @Override
+  @Value.Default
+  default String getType() {
```

Comment:
Is the intention that the `type` can be manually set if needed? If not `@Derived` might be a better fit: https://immutables.github.io/immutable.html#derived-attributes

Reply:
Perfect!  That's exactly what I was looking for (but couldn't find it!)